### ISSUE TITLE
HipChat

### ISSUE THREAD

**Issue by withinfocus** [etcinit/phabulous#54]
Would this project be open to accepting PRs around supporting additional chat protocols? We're interested in HipChat instead of Slack, and this could potentially be used for a general XMPP implementation as well.

**Comment by schemar** [comment]
Absolutely! v3 is built in a way where it is easy to add new connectors, e.g. HipChat or Stride.

**Comment by withinfocus** [comment]
With HipChat being EOLed I'm gonna close this one. We'll be retiring our own instance in the coming months.
